Include Psapi.h after Windows.h

kartarsingh-google opened the pull request:
Some of the chromium bots were not happy after the headers inclusion order was changed after clang-format.

Reverting back the orders in which the headers are added, and turning off clang format for the block where headers are added.

Bug: b/409239273

ddiproietto reviewed (approved):
Can you remove the `Welcome to Perfetto!...` text from the commit message, please?

kartarsingh-google commented:
> Can you remove the Welcome to Perfetto!... text from the commit message, please?

Done, I have removed it from the comment in this thread, I assume it wasn't present in the commit message, but not sure about that, maybe it disappeared from there only after I removed it from the first comment.
